controllers/components/kueue/kueue.go

```diff
@@ -72,3 +74,11 @@ func (s *componentHandler) Init(platform cluster.Platform) error {
 
 	return nil
 }
+
+func (s *componentHandler) GetStatus(ctx context.Context, cli client.Client) (components.Status, error) {
+	k := &componentsv1.Dashboard{}
```

Comment:
```suggestion
	k := &componentsv1.Kueue{}
```

Reply:
I know it's still a draft PR and not yet ready for review, but I thought it might be useful to highlight these in case you didn't catch them before making it ready :slightly_smiling_face:

Reply:
sure, thanks!
history proves this again: do not eat and work at the same time!